torch/_inductor/decomposition.py

```diff
@@ -112,6 +113,8 @@
         aten.softplus,
         aten.softplus_backward,
         aten.bucketize,
+        aten.zero,
```

Comment:
there are already lowerings for aten.zero and aten.zero_ that lower dirrectly to LoopIR and set fill value as constant, I doubt this would be more efficient

Reply:
Right enough, missed those.